.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: go
 
 go:
-  - 1.3
-  - 1.4
+  - 1.4.3
```

Comment:
I was under the impression `1.4` would default to the highest version on TravisCI. Did you change this because `1.4` is `1.4.0`?